test/ArbitrumCrosschainTest.t.sol

```diff
@@ -27,31 +47,9 @@ contract ArbitrumCrosschainTest is CrosschainTestBase  {
         ));
 
         hostDomain.selectFork();
-        forwarder = address(new CrosschainForwarderArbitrum(bridgeExecutor));
         vm.deal(
             L1_EXECUTOR,
             0.01 ether
         );
     }
-
-    function testFuzz_arbitrumGasCalculations(uint256 payloadDataLength) public {
```

Comment:
Looks like theres no test coverage of this in xchain helpers, am I missing something?

Reply:
Taken priv.